examples/onpremise/gettingStartedWeb.php

```diff
@@ -75,57 +75,11 @@
  *
  * ## Class
  */
-require(__DIR__ . "/exampleUtils.php");
+
 require(__DIR__ . "/../../vendor/autoload.php");
```

Comment:
also here?

Reply:
Same as the other examples. They're not part of the package which would be used by client code, so they're not covered by the autoloader.